tests/functional/spec/supplementary_data_interstitial/supplementary_data_interstitial.spec.js

```diff
@@ -0,0 +1,136 @@
+import { click, assertSummaryTitles } from "../../helpers";
```

Comment:
There are many other things tested here, except interstitial, calculated summary, dynamic answers, view submitted response feature. Even the survey name gives you a hint that is happening here: "est_supplementary_data_with_interstitial_and_calculated_summary.json".